installer/dual_rootfs_device_test.go

```diff
@@ -20,6 +20,7 @@ import (
 	"testing"
 	"time"
 
+	"github.com/mendersoftware/mender/conf"
```

Comment:
This file needs its copyright year updated at the very top.

Reply:
Hehe, shrewd. The year is compared against the author date of the latest commit, so you might need `git commit --amend --reset-author` to reset it in 2023.